python/pysrc/kaskada/_timestream.py

```diff
@@ -1192,6 +1194,69 @@ def run_iter(
 
         raise AssertionError(f"Unhandled kind {kind}")
 
+    def plan(
+        self,
+        kind: Literal["initial_dfg", "final_dfg", "final_plan"] = "final_plan",
+        results: Optional[Union[kd.results.History, kd.results.Snapshot]] = None,
+        format: Literal["dot", "svg", "svg_str"] = "svg",
```

Comment:
We may want this to be optional, and then say the default depends on environment / available packages. Eg., I could see defaulting to `dot` if you don't have the pydot package, etc. At which point the logs below could be warnings.

Reply:
@jordanrfrazier any thoughts on whether the format should depend on which packages / environment? I could see a case for not defaulting to `svg` since that only works in IPython notebooks. On the other hand, when I'm in such a notebook I'd really like that to be the default.

Reply:
I wasn't sure the difficulties involved in determining what packages are installed. I think it would be fine to default to `svg`, especially since we're still mainly showing demos and working in notebooks, though.

Reply:
Will be fixed by switch to graphviz package.